### DISCUSSION TITLE
PIP and PBP brightness values don't update for both sliders

### DISCUSSION THREAD

**Discussion by svenoaks** [MonitorControl/MonitorControl#1565] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

If you have a monitor connected to two outputs on the Mac and set to Picture by Picture (and I'd assume PiP, but I haven't actually tried that), there are two separate sets of sliders, one for each of the pictures, which is fine since macOS thinks they are two separate displays. If you slide the brightness slider on one, the brightness for the entire monitor changes including both pictures, again this is fine since I don't think any monitors would have independent brightness for each picture. 

The problem is the slider position for the other picture in MonitorControl is not updated, so you're left with the old brightness value, I'm assuming until the Mac wakes from sleep or another event which causes the brightness to be read from the monitor to happen.

I'd assume this would apply to the volume slider as well (my monitor doesn't have speakers so can't verify).

### Steps to reproduce

1. Plug monitor into the Mac with two connections and set the monitor in PBP or PiP mode.
2. Slide brightness slider for one of the monitors.

### Expected behavior

The brightness and volume sliders slide in unison, or the correct values for the second picture are set after the user stops interacting with the slider.

### Anything else?

I'd be willing to try to fix this and submit a pull request, but I'd like to first hear some input from the developer on the best ways of going about it.

If there was a way of detecting the monitors are actually part of PBP or PIP, the sliders could operate in unison, if not maybe poll all monitors for brightness after user stops interacting with slider?

### Environment Information (please complete the following information)

```markdown
- macOS version: 14.5
- Mac model: MPB late 2023 16"
- MonitorControl version: v4.2.0 build 7048
- Monitor(s): Dell U2722D in PBP mode
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
If the display supports DDC read properly, the app can attempt reading current DDC values. See `Preferences > General > Upon startup or wake`.

You can configure advanced syncing among displays using display groups in https://github.com/waydabber/BetterDisplay, that might help with syncing the sliders.
